.docs/Get-VSTeamProcess.md

```diff
@@ -50,7 +62,10 @@ Default value: 100
 
 ### -Skip
 
-Defines the number of Process Templates to skip. The default value is 0
+Defines the number of Process Templates to skip.
```

Comment:
Update the help to say this is deprecated and going to be removed.

Reply:
Will do.